pkg/reconciler/certificate/certificate.go

```diff
@@ -107,6 +108,8 @@ func (c *Reconciler) reconcile(ctx context.Context, knCert *v1alpha1.Certificate
 	// Propagate cert-manager Certificate status to Knative Certificate.
 	cmCertReadyCondition := resources.GetReadyCondition(cmCert)
 	logger.Infof("cm cert condition %v.", cmCertReadyCondition)
+	cmCertRenewingCondition := resources.GetRenewingCondition(cmCert)
```

Comment:
I wonder that it would be better to kick the reconciliation by if `knCert.Status.NotAfter` was changed? (currently L#107 just copied but I think we should check the diff.).

The `GetRenewingCondition` also works but it can surely reconcile the knCert after renewed, I think.